uint8 mode  # 0=pid, 1=throttle/duty
float32 angular_velocity_left # rad/s
float32 angular_velocity_right # rad/s
float32 duty_left # % max output
float32 duty_right # % max output